src/plone/restapi/tests/test_types.py

```diff
@@ -379,6 +379,31 @@ def test_choice(self):
             adapter.get_schema(),
         )
 
+    def test_choice_inline_array(self):
+        field = schema.Choice(
+            __name__="myfield",
+            title=u"My field",
+            description=u"My great field",
+            values=["foo", "bar"],
+        )
+
+        adapter = getMultiAdapter(
+            (field, self.portal, self.request), IJsonSchemaProvider
+        )
+
+        self.assertEqual(
```

Comment:
It is better to use `assertDictEqual`.
Not a blocker, just a suggestion for the next time ;)